edb/edgeql/compiler/func.py

```diff
@@ -98,7 +98,10 @@ def compile_FunctionCall(
     matched = polyres.find_callable(funcs, args=args, kwargs=kwargs, ctx=ctx)
     if not matched:
         raise errors.QueryError(
-            f'could not find a function variant {funcname}',
+            f'function {funcname} exists, but not with this input',
```

Comment:
This is only marginally better.  We should show a full function signature that we failed to find, like Postgres does:

`function len(integer) does not exist`

This gives more context, i.e. the actual parameter types we tried to match.  We can use the `hint` field to show existing overloads, if any.